platform/mellanox/docker-syncd-mlnx-rpc/Dockerfile.j2

```diff
@@ -45,9 +46,10 @@ RUN apt-get update \
  && cd ..               \
  && rm -fr nanomsg-1.0.0 \
  && rm -f 1.0.0.tar.gz  \
- && pip install cffi==1.7.0    \
- && pip install --upgrade cffi==1.7.0 \
- && pip install nnpy    \
+ && pip2 install cffi==1.7.0    \
+ && pip2 install --upgrade cffi==1.7.0 \
+ && pip2 install wheel  \
+ && pip2 install nnpy    \
```

Comment:
is this change only related to mellanox syncd rpc?

I think it should be applicable to all syncd-rpc

Reply:
Looks like this part is the same among all vendors. So it's likely to be applicable to all syncd-rpc dockers.
As they are defined on a per-vendor basis I think separate PRs should be opened for each vendor.

Reply:
can you fix all the vendors? later, we can unified the rpc docker for all vendors (this can be a separate pr).

Reply:
I can modify the code for all the vendors but I’m not able to compile and test. Is it ok?

Reply:
yes

Reply:
there seems some docker is still stretch based, does it affect by this issue? why is this mellanox rpc docker is broken? what was the trigger?

Reply:
1. The Python has been migrated to version 3
2. cffi build failed because neither Python2-dev nor python3-dev is installed.
3. As ptf_nn_agent and sdk API are using Python2, we need to install those debs for Python 2.

Reply:
Hi @lguohan , 
Could you please help check whether this issue also occurred on other vendor's RPC docker?
This can be done by checking
- whether there is an error when installing nnpy in docker-<vendor>-mlnx-rpc.gz.log, or
- whether the ptf_nn_agent is running.
If it does, I suggest having this bug fixed by this PR.
Thanks

Reply:
i believe this is because of python3 migration. can you check the syncd docker of the platform is buster-based or not? if it is buster based, then we need your patch. if it is stretch based, then we do not need this patch.

Reply:
You’re correct. Only Broadcom and we use buster based syncs rpc docker.
code updated accordingly.